realm/realm-library/build.gradle

```diff
@@ -236,23 +236,32 @@ task pmd(type: Pmd) {
     }
 }
 
+// Configure Checkstyle
+// Android sourceSets are not sourceSets, so we can't confgure this with the DSL.
 task checkstyle(type: Checkstyle) {
-    group = 'Test'
+    group = 'Verification'
 
     source 'src'
-    include '**/*.java'
-    exclude '**/gen/**'
-    exclude '**/R.java'
-    exclude '**/BuildConfig.java'
+    include '*/java/**/*.java'
+    exclude 'benchmarks/**'
+    // Ingore tests for now.
+    exclude '*Test*/**'
 
-    def configProps = ['proj.module.dir': projectDir.absolutePath]
-    configProperties configProps
+    // empty classpath
+    classpath = files()
+}
+
+checkstyle {
+    toolVersion ="7.6"
 
     configFile = file("${projectDir}/../config/checkstyle/checkstyle.xml")
 
-    // empty classpath
-    classpath = files()
+    def configProps = ['proj.module.dir': projectDir.absolutePath]
+    configProperties configProps
+
+    ignoreFailures = true
```

Comment:
Heh!  I think it just does this:`
configProperties configProps = ['proj.module.dir': projectDir.absolutePath]
`
I copied it over, exactly, from the pre-existing code, 'cause I didn't understand it, either!